docker-compose.demo.yml

```diff
@@ -54,7 +54,7 @@ services:
       container_name: flower
       image: layman:latest
       user: ${UID_GID}
-      command: bash -c "cd src && python3 wait_for_deps.py && python3 -m flower -A layman.celery_app --port=5555 --url_prefix=flower"
+      command: bash -c "cd src && python3 wait_for_deps.py && celery -A layman.celery_app flower --url_prefix=flower"
```

Comment:
Strage that port=5555 is in not demo and it is in dev. Try what combinations work